cpp/logger/SFAwsLogger.hpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018 Snowflake Computing, Inc. All rights reserved.
+ */
+
+#ifndef SNOWFLAKECLIENT_SFAWSLOGGER_HPP
+#define SNOWFLAKECLIENT_SFAWSLOGGER_HPP
+
+#include <aws/core/utils/logging/LogSystemInterface.h>
+
+using Aws::Utils::Logging::LogLevel;
+
+namespace Snowflake
+{
+namespace Client
+{
+namespace Logger
```

Comment:
Do we have multiple logger? If not, we don't need to have this namespace.

Reply:
Ok will remove this one.